src/uu/ln/src/ln.rs

```diff
@@ -126,7 +126,7 @@ static ARG_FILES: &str = "files";
 pub fn uumain(args: impl uucore::Args) -> UResult<()> {
     // clap requires a 'static string
```

Comment:
This is no longer true since clap 4 (and the implementation seems to have changed in the meantime to not have a static str anymore because format returns a `String`).